dashboard/db/migrate/20230518155319_create_parental_permission_requests.rb

```diff
@@ -0,0 +1,12 @@
+class CreateParentalPermissionRequests < ActiveRecord::Migration[6.1]
+  def change
+    create_table :parental_permission_requests do |t|
+      t.references(:user, null: false, type: :integer)
+      t.string :parent_email, null: false
+      t.string :uuid, null: false
```

Comment:
Should UUID be indexed since we will be querying this table looking for the UUID entry when a parent clicks the confirmation link?

Reply:
Yep! Let me add that in